examples/source/interactivity-dynamic-content/Infinite_Cards_Email.html

```diff
@@ -0,0 +1,100 @@
+<!---
+
+preview: amp4email
+author: fstanis
+
+--->
+
+<!--
+## Introduction
+
+This sample demonstrates how to display a feed of data, allowing the user to go through
+a large number of items in an email without having to scroll.
+
+The sample uses a combination of [`amp-list`](/documentation/components/amp-list),
+to fetch the initial items from the server and [`amp-form`](/documentation/components/amp-form),
+to "refresh" a single item, by making a new server request.
+-->
+
+<!-- -->
+<!doctype html>
+<html ⚡4email>
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+  <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
+  <style amp4email-boilerplate>body{visibility:hidden}</style>
+
+  <!--
+    ## Styles
+
+    We use CSS to hide the initially fetched item after the form is first submitted.
+
+    We also define a layout that allows us to have fixed card sizes, to ensure form submissions don't result in content jumps.
+  -->
+  <style amp-custom>
+    .amp-form-submit-success .initial-content,
+    .amp-form-submitting .initial-content,
+    .amp-form-submit-error .initial-content {
+      display: none;
+    }
+
+    .card {
+      width: 160px;
+      height: 120px;
+      margin: 10px;
+      float: left;
+      position: relative;
+    }
+
+    .card .next-button {
+      position: absolute;
+      bottom: 0;
+      width: 100%;
+    }
+  </style>
+</head>
+<body>
+  <!--
+    ## Single item template
+
+    Define a template for a single item inside a card and give it an `id`. This template is used by `amp-form` for displaying new items.
```

Comment:
You're changing the way you explain your example from the explanatory "We use ..." to the tutorial style "Define ..". I prefer the first style.